modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -65,23 +65,27 @@ $ oc adm upgrade --to-multi-arch
 $ oc adm upgrade
 ----
 +
+.Example output
+
+[source,terminal]
+----
+working towards ${VERSION}: 106 of 841 done (12% complete), waiting on machine-config
+----
++
 [IMPORTANT]
 ====
 Machine launches may fail as the cluster settles into the new state. To notice and recover when machines fail to launch, we recommend deploying machine health checks. For more information about machine health checks and how to deploy them, see _About machine health checks_.
 ====
 +
-//commenting this section out until https://issues.redhat.com/browse/OCPBUGS-8256 is resolved:
-//For `oc get co`, expect `AVAILABLE=True`, `PROGRESSING=False`, and `DEGRADED=False` on all cluster Operators.
-+
-//For `oc get mcp`, expect `UPDATED=True`, `UPDATING=False`, and `DEGRADED=False` on all machine config pools.
+* If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps), steps always need to be written in the imperative mood (in this case that would mean removing "you can"). If this is an optional step, then [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps) suggests you start the step with "Optional:" and then still write the rest of the step in the imperative mood, as seen in the examples there.
```suggestion
* Optional: To retrieve more detailed information about the status of your update, monitor the migration by running the following Technology Preview command:
```